examples/webpack/karma.conf.js

```diff
@@ -0,0 +1,88 @@
+const webpack = require('webpack');
+
+const webpackConfig = {
+  module: {
+    loaders: [{
+      test: /\.ts$/,
+      loader: 'ts',
+      exclude: /node_modules/
+    }],
+    postLoaders: [{
+      test: /src\/.+\.ts$/,
+      exclude: /(test|node_modules|\.spec)/,
```

Comment:
Good catch, I think that was left over in my copy pasting this from my projects config. How about `exclude: /(node_modules|\.spec\.ts$)/,`